src/components/Typography.js

```diff
@@ -27,15 +27,15 @@ export const H3 = styled.h3`
 
 export const P = styled.p`
   ${text};
-  ${p => (p.highlight && 'color: ' + p.theme.colors.link)};
+  ${p => (p.highlight && 'color: ' + p.theme.colors.highlight)};
   margin: 0;
 `
 
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  color: ${p => (p.selected ? p.theme.colors.linkHover : p.theme.colors.link)};
-  transition: color 0.3s cubic-bezier(.25,.8,.25,1);
+  color: ${p => p.theme.colors.link};
+  transition: all 0.5s cubic-bezier(.25,.8,.25,1);
```

Comment:
why is this `all`? what else are we transitioning

Reply:
color and background-color